theunderground/movies.py

```diff
@@ -54,11 +54,20 @@ def add_movie():
 
     if form.validate_on_submit():
         movie = form.movie.data
+        dsmovie = form.dsmovie.data
         thumbnail = form.thumbnail.data
         if movie and thumbnail:
             movie_data = movie[0].read()
             thumbnail_data = thumbnail[0].read()
 
+            if dsmovie:
+                dsmovie_data = dsmovie[0].read()
+                ds_mov_id = 2
```

Comment:
We can't hardcode this value because this is how it gets the movie from the server.

Reply:
the hardcoded value was a mistake. what should go there instead?

Reply:
Remove line 65 and refer to my second comment.

Reply:
resolved in 46a2678